service/src/main/resources/application.yml

```diff
@@ -89,7 +89,13 @@ management:
   endpoints:
     web:
       exposure:
-        include: "prometheus"
+        include: "*"
```

Comment:
what else is caught with this "*" now?

Reply:
all of [these](https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html#actuator.endpoints) . I have seen other services do this too but yea im not sure we actually need to expose them all.  They arent publicly exposed so its not a security risk or anything.  If anything it would be a performance thing to think about.

Reply:
would it make sense to add a comment with this link in case we come back in the future and don't remember what this is about? or is it pretty straightforward to find it given the context here, do you think?